server/sequencer_manager.go

```diff
@@ -49,7 +49,7 @@ func (s SequencerManager) Name() string {
 
 // ExecutePass performs sequencing for the specified set of Logs.
 func (s SequencerManager) ExecutePass(logIDs []int64, logctx LogOperationManagerContext) bool {
-	glog.V(1).Infof("Beginning sequencing run for %d active log(s)", len(logIDs))
+	glog.V(1).Infof("beginning sequencing run for %v active log(s)", len(logIDs))
```

Comment:
Done.

Thankfully I didn't do that *everywhere* in a bunch of different PRs. Imagine that... :)